habitat_baselines/il/vln/dagger.py

```diff
@@ -630,20 +643,32 @@ def train(self) -> None:
                                         device=self.device, non_blocking=True
                                     ),
                                 )
+                                loss += output[0]
+                                action_loss += output[1]
+                                aux_loss += output[2]
 
                         logger.info(f"train_loss: {loss}")
+                        logger.info(f"train_action_loss: {action_loss}")
+                        logger.info(f"train_aux_loss: {aux_loss}")
                         logger.info(f"Batches processed: {step_id}.")
                         logger.info(
                             f"On DAgger iter {dagger_it}, Epoch {epoch}."
                         )
                         writer.add_scalar(
                             f"train_loss_iter_{dagger_it}", loss, step_id
                         )
+                        writer.add_scalar(
+                            f"train_action_loss_iter_{dagger_it}", action_loss, step_id
+                        )
+                        writer.add_scalar(
+                            f"train_aux_loss_iter_{dagger_it}", aux_loss, step_id
```

Comment:
In the event we had multiple aux losses, we would likely want to monitor them separately. Getting a dict from `AuxLosses` could do that for us. I don't think we plan on multiple aux losses, just a general thought.

Reply:
Yes, agreed. We should updated this when we have another auxiliary loss.